Checkstyle error on windows

Hi, I can't build the project on windows. It ends with error:

<details>
$ ./gradlew clean build --parallel

> Configure project :
  Building version '2.28.3' (value loaded from 'version.properties' file).

> Task :deprecatedPluginsTest:checkstyleTest FAILED
> Task :android:checkstyleMain FAILED
> Task :errorprone:checkstyleMain FAILED
> Task :junit-jupiter:checkstyleMain FAILED
> Task :inline:checkstyleTest FAILED
w: Detected multiple Kotlin daemon sessions at build\kotlin\sessions

FAILURE: Build completed with 5 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':deprecatedPluginsTest:checkstyleTest'.
> Unable to create Root Module: config {G:\programowanie\forki\mockito\config\checkstyle\checkstyle.xml}, classpath {G:\programowanie\forki\mockito\subprojects\deprecatedPluginsTest\build\classes\java\test;G:\programowanie\forki\mockito\subprojects\deprecatedPluginsTest\build\resources\test;G:\programowanie\forki\mockito\subprojects\deprecatedPluginsTest\build\classes\java\main;G:\programowanie\forki\mockito\subprojects\deprecatedPluginsTest\build\resources\main;G:\programowanie\forki\mockito\build\libs\mockito-core-2.28.3.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\junit\junit\4.12\2973d150c0dc1fefe998f834810d68f278ea58ec\junit-4.12.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\net.bytebuddy\byte-buddy\1.9.10\211a2b4d3df1eeef2a6cacf78d74a1f725e7a840\byte-buddy-1.9.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\net.bytebuddy\byte-buddy-agent\1.9.10\9674aba5ee793e54b864952b001166848da0f26b\byte-buddy-agent-1.9.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.objenesis\objenesis\2.6\639033469776fd37c08358c6b92a4761feb2af4b\objenesis-2.6.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.hamcrest\hamcrest-core\1.3\42a25dc3219429f0e5d060061f71acb49bf010a0\hamcrest-core-1.3.jar}.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':android:checkstyleMain'.
> Unable to create Root Module: config {G:\programowanie\forki\mockito\config\checkstyle\checkstyle.xml}, classpath {G:\programowanie\forki\mockito\subprojects\android\build\classes\java\main;G:\programowanie\forki\mockito\subprojects\android\build\resources\main;G:\programowanie\forki\mockito\build\libs\mockito-core-2.28.3.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\net.bytebuddy\byte-buddy-android\1.9.10\fda8cb1715f7063c6177b0b3e9c09096b13b4bbb\byte-buddy-android-1.9.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\net.bytebuddy\byte-buddy\1.9.10\211a2b4d3df1eeef2a6cacf78d74a1f725e7a840\byte-buddy-1.9.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\net.bytebuddy\byte-buddy-agent\1.9.10\9674aba5ee793e54b864952b001166848da0f26b\byte-buddy-agent-1.9.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.objenesis\objenesis\2.6\639033469776fd37c08358c6b92a4761feb2af4b\objenesis-2.6.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.jakewharton.android.repackaged\dalvik-dx\1\5344672f614ef1df39f7a9d88a5aa0e44d74d4d9\dalvik-dx-1.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.jakewharton.android.repackaged\libcore-dex\2\1d3c98644912295ca8cb80069b1cdea4389085b2\libcore-dex-2.jar}.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

3: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':errorprone:checkstyleMain'.
> Unable to create Root Module: config {G:\programowanie\forki\mockito\config\checkstyle\checkstyle.xml}, classpath {G:\programowanie\forki\mockito\subprojects\errorprone\build\classes\java\main;G:\programowanie\forki\mockito\subprojects\errorprone\build\resources\main;G:\programowanie\forki\mockito\build\libs\mockito-core-2.28.3.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.errorprone\error_prone_core\2.3.2\d5d121a23bcd48df2fe42dc3f1424cd05872993\error_prone_core-2.3.2.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.auto.service\auto-service\1.0-rc5\d25246bae325b4bcc63b55d6d782515fac32215a\auto-service-1.0-rc5.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\net.bytebuddy\byte-buddy\1.9.10\211a2b4d3df1eeef2a6cacf78d74a1f725e7a840\byte-buddy-1.9.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\net.bytebuddy\byte-buddy-agent\1.9.10\9674aba5ee793e54b864952b001166848da0f26b\byte-buddy-agent-1.9.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.objenesis\objenesis\2.6\639033469776fd37c08358c6b92a4761feb2af4b\objenesis-2.6.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.errorprone\error_prone_check_api\2.3.2\7415438c00adec8ba707689ec4168c8484d8403b\error_prone_check_api-2.3.2.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.errorprone\error_prone_annotation\2.3.2\7c554c59dd2ea8c4e9e36b3308f8bf92db83e70c\error_prone_annotation-2.3.2.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.errorprone\error_prone_type_annotations\2.3.2\e79e88b9051888c8ea806f49c86fa7e3e3728180\error_prone_type_annotations-2.3.2.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.github.stephenc.jcip\jcip-annotations\1.0-1\ef31541dd28ae2cefdd17c7ebf352d93e9058c63\jcip-annotations-1.0-1.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.pcollections\pcollections\2.1.2\15925fd6c32a29fe3f40a048d238c5ca58cb8362\pcollections-2.1.2.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.auto\auto-common\0.10\c8f153ebe04a17183480ab4016098055fb474364\auto-common-0.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.guava\guava\27.0.1-jre\bd41a290787b5301e63929676d792c507bbc00ae\guava-27.0.1-jre.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.code.findbugs\jFormatString\3.0.0\d3995f9be450813bc2ccee8f0774c1a3033a0f30\jFormatString-3.0.0.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.code.findbugs\jsr305\3.0.2\25ea2e8b0c338a877313bd4672d3fe056ea78f0d\jsr305-3.0.2.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.checkerframework\dataflow\2.5.3\edf284e0838290d661b22483ecf648065e7ec440\dataflow-2.5.3.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.errorprone\javac\9+181-r4173-1\bdf4c0aa7d540ee1f7bf14d47447aea4bbf450c5\javac-9+181-r4173-1.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.errorprone\error_prone_annotations\2.3.2\d1a0c5032570e0f64be6b4d9c90cdeb103129029\error_prone_annotations-2.3.2.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.protobuf\protobuf-java\3.4.0\b32aba0cbe737a4ca953f71688725972e3ee927c\protobuf-java-3.4.0.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.auto.service\auto-service-annotations\1.0-rc5\6ea999af2b6262a7179a09c51a3d54e7b40a3833\auto-service-annotations-1.0-rc5.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.googlecode.java-diff-utils\diffutils\1.3.0\7e060dd5b19431e6d198e91ff670644372f60fbd\diffutils-1.3.0.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.github.kevinstern\software-and-algorithms\1.0\5e77666b72c6c5dd583c36148d17fc47f944dfb5\software-and-algorithms-1.0.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.checkerframework\javacutil\2.5.3\c545ca6fc7a57e3bc65d46e8e9438376f0db35ea\javacutil-2.5.3.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.checkerframework\checker-qual\2.5.3\4fe154d21bd734fe8c94ada37cdc41a9a6d61776\checker-qual-2.5.3.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.guava\failureaccess\1.0.1\1dcf1de382a0bf95a3d8b0849546c88bac1292c9\failureaccess-1.0.1.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.guava\listenablefuture\9999.0-empty-to-avoid-conflict-with-guava\b421526c5f297295adef1c886e5246c39d4ac629\listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\com.google.j2objc\j2objc-annotations\1.1\ed28ded51a8b1c6b112568def5f4b455e6809019\j2objc-annotations-1.1.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.codehaus.mojo\animal-sniffer-annotations\1.17\f97ce6decaea32b36101e37979f8b647f00681fb\animal-sniffer-annotations-1.17.jar}.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

4: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':junit-jupiter:checkstyleMain'.
> Unable to create Root Module: config {G:\programowanie\forki\mockito\config\checkstyle\checkstyle.xml}, classpath {G:\programowanie\forki\mockito\subprojects\junit-jupiter\build\classes\java\main;G:\programowanie\forki\mockito\subprojects\junit-jupiter\build\resources\main;G:\programowanie\forki\mockito\build\libs\mockito-core-2.28.3.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.junit.jupiter\junit-jupiter-api\5.1.1\fcca73c9c2825c9a966380c6d146a71acba233ca\junit-jupiter-api-5.1.1.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\net.bytebuddy\byte-buddy\1.9.10\211a2b4d3df1eeef2a6cacf78d74a1f725e7a840\byte-buddy-1.9.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\net.bytebuddy\byte-buddy-agent\1.9.10\9674aba5ee793e54b864952b001166848da0f26b\byte-buddy-agent-1.9.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.objenesis\objenesis\2.6\639033469776fd37c08358c6b92a4761feb2af4b\objenesis-2.6.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.junit.platform\junit-platform-commons\1.1.1\36fca464a84454cf56ae2759a8ba18d6d99d1113\junit-platform-commons-1.1.1.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.apiguardian\apiguardian-api\1.0.0\3ef5276905e36f4d8055fe3cb0bdcc7503ffc85d\apiguardian-api-1.0.0.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.opentest4j\opentest4j\1.0.0\6f09c598e9ff64bf0ce2fa7e7de49a99ba83c0b4\opentest4j-1.0.0.jar}.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

5: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':inline:checkstyleTest'.
> Unable to create Root Module: config {G:\programowanie\forki\mockito\config\checkstyle\checkstyle.xml}, classpath {G:\programowanie\forki\mockito\subprojects\inline\build\classes\java\test;G:\programowanie\forki\mockito\subprojects\inline\build\resources\test;G:\programowanie\forki\mockito\subprojects\inline\build\classes\java\main;G:\programowanie\forki\mockito\subprojects\inline\build\resources\main;G:\programowanie\forki\mockito\build\libs\mockito-core-2.28.3.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\junit\junit\4.12\2973d150c0dc1fefe998f834810d68f278ea58ec\junit-4.12.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\net.bytebuddy\byte-buddy\1.9.10\211a2b4d3df1eeef2a6cacf78d74a1f725e7a840\byte-buddy-1.9.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\net.bytebuddy\byte-buddy-agent\1.9.10\9674aba5ee793e54b864952b001166848da0f26b\byte-buddy-agent-1.9.10.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.objenesis\objenesis\2.6\639033469776fd37c08358c6b92a4761feb2af4b\objenesis-2.6.jar;F:\programowanie\gradleCache\caches\modules-2\files-2.1\org.hamcrest\hamcrest-core\1.3\42a25dc3219429f0e5d060061f71acb49bf010a0\hamcrest-core-1.3.jar}.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
==============================================================================

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.3.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 38s
108 actionable tasks: 87 executed, 7 from cache, 14 up-to-date
</details>

I've fixed this issue but just need an issue for PR.

--- a/config/checkstyle/checkstyle.xml
+++ b/config/checkstyle/checkstyle.xml
@@ -28,7 +28,7 @@
         </module>
     </module>
     <module name="RegexpHeader">
-        <property name="headerFile" value="config/checkstyle/java.header"/>
+        <property name="headerFile" value="${config_loc}/java.header"/>
         <property name="fileExtensions" value="java"/>
     </module>
 </module>

--- a/src/test/java/org/mockitousage/matchers/MatchersTest.java
+++ b/src/test/java/org/mockitousage/matchers/MatchersTest.java
@@ -608,7 +608,7 @@ public class MatchersTest extends TestBase {
     public void nullable_matcher() throws Exception {
         // imagine a Stream.of(...).map(c -> mock.oneArg(c))...
         mock.oneArg((Character) null);
-        mock.oneArg(Character.valueOf('€'));
+        mock.oneArg(Character.valueOf('\u20AC'));
 
         verify(mock, times(2)).oneArg(nullable(Character.class));
     }

